engine/settings.cloud9.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * Defines database credentials.
+ *
+ * Most of Elgg's configuration is stored in the database.  This file contains the
+ * credentials to connect to the database, as well as a few optional configuration
+ * values.
+ *
+ * The Elgg installation attempts to populate this file with the correct settings
+ * and then rename it to settings.php.
+ *
+ * @todo Turn this into something we handle more automatically.
+ * @package    Elgg.Core
+ * @subpackage Configuration
+ */
+
+global $CONFIG;
+if (!isset($CONFIG)) {
+	$CONFIG = new \stdClass;
+}
+
+// Must set timezone explicitly for cloud9!
+date_default_timezone_set('America/Los_Angeles');
```

Comment:
Does it matter? Cloud9 is a dev server. This file probably shouldn't even be here.